docs/docsite/rst/windows_winrm.rst

```diff
@@ -657,6 +666,34 @@ for each authentication option. See the section on authentication above for more
     encryption done over TLS. The WinRM payload is still encrypted with TLS
     when run over HTTPS, even if ``ansible_winrm_message_encryption=never``.
 
+HTTPS Certificate Validation
+````````````````````````````
+As part of the HTTPS protocol, the certificate is validated to ensure the host
+matches the subject and the client trusts the issuer of the server certificate.
+When using a self-signed certificate or setting
+``ansible_winrm_server_cert_validation: ignore`` these security mechanisms are
+bypassed. While self signed certificates will always need the ``ignore`` flag,
+certificates that have been issued from a certificate authority can still be
+validated.
+
+One of the more common ways of setting up a HTTPS listener in a domain
+environment is to use Active Directory Certificate Service (AD CS). AD CS is
+used by Active Directory to sign certificate requests that are sent to it and
+then issue a certificate from the csr. The issuer certificate of the AD CS
```

Comment:
Explain what a “CSR” is. In general, define abbreviations on first use and then you can just use the abbreviation (capitalized, please) for subsequent mentions.